packages/desktop-client/src/components/tooltips.tsx

```diff
@@ -111,7 +157,7 @@ export class Tooltip extends Component {
     const { ignoreBoundary = false } = this.props;
 
     if (!ignoreBoundary && this.context) {
-      return this.context.current;
+      return (this.context as RefObject<HTMLElement>).current;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏if you set `context: RefObject<HTMLElement>` (below `target`) on the root level you should be able to get rid of this manual type coercion.

Reply:
So for this I originally had `declare context: RefObject<HTMLElement>;`, seemed like the declare keyword was causing tests to fail, when researching a fix babel + webpack changes were required which is why I went for the above instead. 
Here is the error: https://github.com/actualbudget/actual/actions/runs/7184512922/job/19565740541?pr=2073#step:4:135

Your suggestion without the declare keyword produces the following error when running typecheck
```
packages/desktop-client/src/components/tooltips.tsx:76:3 - error TS2612: Property 'context' will overwrite the base property in 'Component<TooltipProps, {}, any>'. If this is intentional, add an initializer. Otherwise, add a 'declare' modifier or remove the redundant declaration.

76   context: RefObject<HTMLElement>;
     ~~~~~~~


Found 1 error in packages/desktop-client/src/components/tooltips.tsx:76
```

Reply:
It's been a while I last used class-components. We're actually slowly moving everything over to functional.

Nonetheless, would adding a `!` solve the problem? Any other alternatives? I agree that changing webpack/babel configs here would not be ideal.

```
context!: RefObject<HTMLElement>
```

Reply:
I added an initializer which resolved this issue. Let me know what  you think